lib/dkdeploy/interaction_handler/mysql.rb

```diff
@@ -0,0 +1,37 @@
+module Dkdeploy
+  module InteractionHandler
+    # Interaction handler for sending password to MySQL client
+    # This InteractionHandler provides output of the error code if MySQL
+    # answers with an error to the command.
+    #
+    # @attr [String] password The password to send to terminal
+    class MySql
+      def initialize(password)
+        @password = password
+        # these two are declared as instance variables because the on_data method is called multiple times
+        @return_message = ''
+        @mysql_error_seen = false
+      end
+
+      # Method to send password to terminal
+      #
+      # @param [SSHKit::Command] _command
+      # @param [Symbol] _stream_name
+      # @param [String] data
+      # @param [Net::SSH::Connection::Channel] channel
+      def on_data(_command, _stream_name, data, channel)
+        if data =~ /.*password.*/i
+          channel.send_data("#{@password}\n")
+        else
+          @mysql_error_seen = true if data =~ /.*ERROR.*/i
```

Comment:
Do we need this logik at this point?
Can we simplify that and output all the stream information here?
What types of information can happen here?